projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.component.ts

```diff
@@ -88,6 +88,11 @@ export class NgxSkeletonLoaderComponent implements OnInit, AfterViewInit, OnDest
       }
       this.count = 1;
     }
+
+    //Force count to 1 when customContent is used
+    if (this.appearance === 'customContent'){
```

Comment:
Could you apply also a `console.error()` message on devMode, please? So that, we can spread decisions we've done on the component level at this stage.

```suggestion
if (this.appearance === 'custom-content'){
	// Shows error message only in Development
	if (isDevMode() && this.count !== 1) {
		console.error(
			`\`NgxSkeletonLoaderComponent\` enforces elements with "custom-content" appearance as DOM nodes. Forcing "count" to "1".`,
		);
		this.count = 1;
	}
}
```